src/app/services/baw-api/audio-event-import-file/audio-event-import-file.service.ts

```diff
@@ -94,20 +102,89 @@ export class AudioEventImportFileService
     audioEventImport: AudioEventImport,
     provenanceId: Id,
   ) {
-    const params: { commit?: boolean, provenance_id?: number } = {};
+    const params: { commit?: boolean; provenance_id?: number } = {};
     if (provenanceId !== null) {
       params.provenance_id = provenanceId;
     }
 
     // we don't want to raise non-200 responses as errors because the api will
     // respond with a 422 (unprocessable content) error if a dry run fails
-    return this.api
-      .create(
-        AudioEventImportFile,
-        endpoint(audioEventImport, emptyParam, emptyParam),
-        (event) => endpoint(audioEventImport, event, emptyParam),
-        model,
-        { disableNotification: true, params }
-      );
+    return this.api.create(
+      AudioEventImportFile,
+      endpoint(audioEventImport, emptyParam, emptyParam),
+      (event) => endpoint(audioEventImport, event, emptyParam),
+      model,
+      { disableNotification: true, params },
+    );
+  }
+}
+
+// TODO: Refactor this service to use the API once we have a shallow audio event
+// import file routes.
+// see: https://github.com/QutEcoacoustics/baw-server/issues/871
+@Injectable()
+export class ShallowAudioEventImportFileService
+  implements
+    StandardApi<AudioEventImportFile, [CollectionIds<AudioEventImport>]>
+{
+  // TODO: Replace this with the BawApiService once shallow routes are available
+  private readonly api = inject(AudioEventImportFileService);
+
+  public list(): Observable<AudioEventImportFile[]> {
+    throw new Error("Method not implemented.");
+  }
+
+  public show(
+    _model: IdOr<AudioEventImportFile>,
+  ): Observable<AudioEventImportFile> {
+    throw new Error("Method not implemented.");
+  }
+
+  public create(
+    _model: AudioEventImportFile,
+  ): Observable<AudioEventImportFile> {
+    throw new Error("Method not implemented.");
+  }
+
+  public update(
+    _model: AudioEventImportFile,
+  ): Observable<AudioEventImportFile> {
+    throw new Error("Method not implemented.");
+  }
+
+  public destroy(
+    _model: IdOr<AudioEventImportFile>,
+  ): Observable<void | AudioEventImportFile> {
+    throw new Error("Method not implemented.");
+  }
+
```

Comment:
how about we just don't implement StandardAPI if we're not going to use it.

even if we do shallow we're probably not going to impl all the crud methods, just filter